Riot/Modules/Room/Views/InputToolbar/RoomInputToolbarView.swift

```diff
@@ -20,8 +20,9 @@ import GBDeviceInfo
 
 extension RoomInputToolbarView {
     open override func sendCurrentMessage() {
-        // Triggers auto-correct if needed.
-        if self.isFirstResponder {
+        // Triggers auto-correct if needed and if it is not a command.
+        let isCommand = self.textMessage.hasPrefix("/")
+        if self.isFirstResponder && !isCommand {
```

Comment:
I think the issue with that code is that it will only avoid triggering that last auto-correction that happens upon sending the message, and it will still still trigger upon typing. 
I suppose we could do something similar that what the RTE does if it works: https://github.com/matrix-org/matrix-rich-text-editor/blob/main/platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Extensions/UITextView.swift